app/code/Magento/User/Controller/Adminhtml/Auth.php

```diff
@@ -6,6 +6,7 @@
 namespace Magento\User\Controller\Adminhtml;
 
 use Magento\Framework\Encryption\Helper\Security;
+use Magento\Framework\App\ObjectManager;
```

Comment:
Since you are refactoring, please add import statement for all used classes in this file.

Reply:
Done.

Reply:
Thanks! Please also do the ones on line 24  and 29.

Reply:
Done.